sync/class.jetpack-sync-listener.php

```diff
@@ -173,6 +173,15 @@ function enqueue_action( $current_filter, $args, $queue ) {
 			return;
 		}
 
+		/**
+		 * Add an action hook to execute when anything on the whitelist gets sent to the queue to sync.
+		 *
+		 * @since 5.8
+		 *
```

Comment:
Could you add `@module sync` here? We use this to create categories in our codex:
https://developer.jetpack.com/module/sync/